.python-version

```diff
@@ -0,0 +1 @@
+marketplace
```

Comment:
Assuming integration with `pyenv`, this merely indicates the desired Python virtual environment, which is then automatically loaded upon `cd` into the project directory; (and, `develop` gets the name of the environment it creates from this file).